presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/Partition.java

```diff
@@ -56,7 +58,8 @@ public Partition(
             @JsonProperty("partitionVersion") Optional<Long> partitionVersion,
             @JsonProperty("eligibleToIgnore") boolean eligibleToIgnore,
             @JsonProperty("sealedPartition") boolean sealedPartition,
-            @JsonProperty("createTime") int createTime)
+            @JsonProperty("createTime") int createTime,
+            @JsonProperty("lastAccessTime") DateTime lastAccessTime)
```

Comment:
Can we use integer timestamp instead of a complex data structure to be consistent with createTime? It should be more efficient and easier to handle in the downstream pipelines.

Reply:
Yeah, good point. @highker may have different point on this.